### PR TITLE
PS-8680 Restructure table of contents (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
modified:   mkdocs-base.yml

### CODE REVIEW

**Review by alina-derkach-oaza — APPROVED** [review]

File: docs/quickstart-overview.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Quickstart Overview
```
Comment (alina-derkach-oaza) [inline docs/quickstart-overview.md]: Quickstart Overview -> Quickstart overview

File: mkdocs-base.yml [hunk 2]
```diff
@@ -121,183 +121,220 @@ extra:
...
+       - quickstart-overview.md
+  - Install:
+      - Install Percona Server for MySQL from repositories: installation.md
       - Use APT:
-        - Install using APT repository: installation/apt-repo.md
+        - Use APT repositories: installation/apt-repo.md       
         - Files in DEB package: installation/apt-files.md
         - installation/build-apt-packages.md
-        - Install from downloaded DEB packages: installation/apt-download-deb.md
+        - Downloaded DEB packages: installation/apt-download-deb md
```
Comment (alina-derkach-oaza) [inline mkdocs-base.yml]: installation/apt-download-deb md -> add a dot before md

File: docs/quickstart-overview.md (hunk 1 above)
Comment (patrickbirch) [reply docs/quickstart-overview.md]: Done.

File: mkdocs-base.yml (hunk 2 above)
Comment (patrickbirch) [reply mkdocs-base.yml]: Added dot

**Pull request merged**